src/ocm/hooks/useOpenshiftVersions.tsx

```diff
@@ -25,14 +25,16 @@ export default function useOpenshiftVersions(): UseOpenshiftVersionsType {
   const doAsync = React.useCallback(async () => {
     try {
       const { data } = await SupportedOpenshiftVersionsAPI.list();
-      const versions: OpenshiftVersionOptionType[] = Object.keys(data).map((key) => ({
-        label: `OpenShift ${data[key].displayName || key}`,
-        value: key,
-        version: data[key].displayName,
-        default: Boolean(data[key].default),
-        supportLevel: data[key].supportLevel,
-        cpuArchitectures: data[key].cpuArchitectures as CpuArchitecture[],
-      }));
+      const versions: OpenshiftVersionOptionType[] = Object.entries(data).map(
```

Comment:
According to the issue, the type `OpenshiftVersionOptionType` could be amended given that `cpuArchirecture` is a required field.

Reply:
I couldn't change it since this type is used also by CIM. 
I'll create another type for OCM

Reply:
actually creating another type for OCM will involve a lot of code changes... not sure it's worth it

Reply:
I hadn't thought of that. It's not a big deal, so feel free to leave as is :)

Reply:
can we mark the thread as resolved?